dist/enhance.css

```diff
@@ -126,6 +126,11 @@ svg,
 video {
   display: block;
   max-width: 100%;
+}
+img,
+picture,
+svg,
+video {
```

Comment:
This addition comes from a previous change in which the distribution file wasn't regenerated